src/ClientApp/src/app/voters-guide/voters-guide.component.scss

```diff
@@ -1,49 +1,90 @@
 .wrapper {
-  align-items: center;
-  text-align: center;
-  margin-top: 50px;
-
-  p {
-    font-style: normal;
-    font-weight: 600;
-  }
-
-  .options-list {
-    width: 100%;
-    display: flex;
-    flex-wrap: wrap;
-
-    .option {
-      background: #ffcc00;
-      color: #352245;
-      width: 200px;
-      height: 100px;
-      margin: 5px;
-      font-weight: 600;
-      font-size: 16px;
-      line-height: 24px;
-      padding: 5px;
-      display: flex;
-      align-items: center;
-      justify-content: center;
-
-      &.is-selected {
-        background: #352245;
-        color: #ffcc00;
-      }
-    }
-  }
-
-  @media (max-width: 575px) {
-    .tile-list {
-      display: block;
-    }
-
-    .options-list {
-      .option {
-        width: 48%;
-        margin: 1%;
-      }
-    }
-  }
+	align-items: center;
+	margin-top: 50px;
+	p {
+		font-style: normal;
+		font-weight: 600;
+	}
+	.options-list {
+		width: 100%;
+		display: flex;
+		flex-wrap: wrap;
+		.option {
+			text-align: center;
+			position: relative;
+			color: #352245;
+			width: 215px;
+			height: 100px;
+			margin: 5px;
+			font-weight: 600;
+			font-size: 16px;
+			line-height: 24px;
+			padding: 5px;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			border: #352245 solid 1px;
+			overflow: hidden;
+			.check-icon-container {
+				width: 45px;
+				height: 45px;
+				background-color: #352245;
+				position: absolute;
+				top: 0;
+				right: 0;
+				transform: translateX(50%) translateY(-50%) rotate(45deg);
+				transition: all 0.2s ease-in-out;
+				opacity: 0;
+			}
+			.check-icon {
+				opacity: 0;
+				display: hidden;
```

Comment:
`hidden` is an invalid value for `display` property. Use either `display: none` or  `visibility: hidden`.

In both cases, the `opacity: 0` above is redundant.

Reply:
this property `opacity: 0` is afected by `transition: all` going from 1..0 and vice-versa so it should stay

Reply:
`display: none` will hide the element completely. Changing it's opacity won't have any effect.